US12789: read time

lukemoderwell opened the pull request:
PR adds:
- read time calculated by taking word count and dividing by 184 (avg words per min read)
- include for read time markup
- abstract article card to an include

tcmacdonald commented on _includes/_article-card.html:
```diff
@@ -0,0 +1,21 @@
+    <div class="card-bgImage sixteen-nine" style="background-image: url('{{ post.featured_image }}');"></div>
+    <div class="card-block">
+        <h5 class="publication_meta metadata flush">
+        {% assign words = post.content | number_of_words %}
```
Can this be refactored to use the `read-time.html` partial?

lukemoderwell replied:
This was another case of `page` vs `post`. If you know a good way to programmatically switch between those, let me know.

tcmacdonald commented on _includes/_article-body.html:
```diff
@@ -1,9 +1,7 @@
 <article class="publication_body article font-family-base">
   <div class="publication_hero push-ends">
     <img src="{{ page.featured_image }}" alt="{{ page.featured_image.title }}" title="" style="width: 100%; height: 500px; object-fit: cover;" />
```
Can we get rid of these styles?

lukemoderwell commented:
@tcmacdonald made some changes to pass the content type in as a variable for the read time include. 
Changed the hero image to a background img to give us a little more control and eliminated the inline styles (other than the background-image). Also changed `publication` to `article` where it made sense. This was carried over from Maestro.

tcmacdonald commented on index.html:
```diff
@@ -18,24 +18,7 @@ <h1 class="section-header text-white">{{ page.heading }}</h1>
-          </div>
-        </a>
-      </div>
+      {% include _article-card.html %}
```
And again, again.
